### PR TITLE
Feature/can 42 safe remaining mobile responsive tasks

### PR DESCRIPTION
**Pull request by 0xAryanShah**
## What it solves
Fixed All the Hidden Button Style,
Create Safe Page : Successful msg fixed.

# Pending
Making Address Smaller like :-  ( 0xab3.......yh8w )

### CODE REVIEW

**Comment by 0xtemujin** [comment]
image

Transactions -> History -> Filter -> Module based -> Click on dropdown : Fix color

**Comment by 0xtemujin** [comment]
image
Theres an invisible button to the left of Download, fix colors

**Comment by 0xtemujin** [comment]
image
Address modals - fix color for Canto

**Comment by 0xAryanShah** [comment]
# Resolves
[Transaction Page] : Tx Hash from Transaction Hash 
[Settings> Owners Page] : Adjust Table
[Transactions/History Page] : the text can be in line to look continuous
[Address Book] : Table has to be adjusted
[Global] : Remove 2 components left bottom	
Cancel button hidden on export modal
Same modal white canto.

## Remaining
Transactions -> History -> Filter -> Module/Import/export based -> Click on dropdown color change
Address Book Make Buttons below Address

**Comment by 0xAryanShah** [comment]
# Resolves
Transactions -> History -> Filter -> Module/Import/export based -> Click on dropdown color change
Address Book Table Adjustment

**Comment by tushar994** [comment]
run `yarn add react-use` and push the changes made by that once? on freshly opening this branch and running `yarn install; yarn start`, the app throws the following error `Module not found: Can't resolve 'react-use' in '/Users/tusharjain/blockchain_work/safe-react/src/routes/safe/components/AddressBook'`. 

Not a big issue because when merged into `dev`, since `dev`'s package.json has `"react-use": "^17.4.0"`, it will work. But still in the interest of cleanliness, maybe make this change?

File: src/components/AppLayout/Sidebar/DebugToggle/index.tsx [hunk 1]
```diff
@@ -11,24 +9,25 @@ const StyledContainer = styled.div`
 `
 
 const DebugToggle = (): React.ReactElement => {
-  const [enabled = false, setEnabled] = useCachedState<boolean>(LS_USE_PROD_CGW)
+  //const [enabled = false, setEnabled] = useCachedState<boolean>(LS_USE_PROD_CGW)
```
Comment (0xtemujin) [inline src/components/AppLayout/Sidebar/DebugToggle/index.tsx]: If not required, remove code don't leave it as commented

File: src/components/AppLayout/Sidebar/index.tsx [hunk 2]
```diff
@@ -123,11 +123,11 @@ const Sidebar = ({
         </>
       ) : null}
 
-      <HelpContainer>
+      {/* <HelpContainer>
```
Comment (0xtemujin) [inline src/components/AppLayout/Sidebar/index.tsx]: If not required remove code, not leave it as commented

**Review by 0xtemujin — CHANGES_REQUESTED** [review]

**Review by 0xtemujin — APPROVED** [review]

**Pull request merged**